configure

```diff
@@ -1648,7 +1650,6 @@ Optional Packages:
   --with-ldap             build with LDAP support
   --with-bonjour          build with Bonjour support
   --with-selinux          build with SELinux support
-  --with-libseccomp       build with libseccomp support
```

Comment:
why remove this option？

Reply:
I'll add this option to configure.ac and re-autoconf.